### PR TITLE
SDK-834 remove symfony 6 packages

### PR DESCRIPTION
**Pull request by sergeyspryker**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
         "doctrine/orm": "^2.11",
         "guzzlehttp/guzzle": "^7.4",
         "monolog/monolog": "^2.3",
-        "ondrejmirtes/better-reflection": "^4.3.81",
+        "ondrejmirtes/better-reflection": "4.3.81",
```
Comment (Kyrylo-Meretskyi) [inline composer.json:28]: Why do we need this change?

Comment (sergeyspryker) [reply composer.json:28]: Because next version `4.4.0` has a constraint `"php": ">=7.4.1,<7.5.0",` that blocks us to use it with php8

**Review by Kyrylo-Meretskyi — APPROVED** [review]

File: composer.json (hunk 1 above)
Comment (pavelmaksimov25) [reply composer.json]: What if we use 5.3 version https://packagist.org/packages/ondrejmirtes/better-reflection#5.3.0?

**Comment by pavelmaksimov25** [comment]
Please provide the ticket number in the description

File: composer.json (hunk 1 above)
Comment (sergeyspryker) [reply composer.json]: Updated the major version

Comment (pavelmaksimov25) [reply composer.json]: I've checked b2b demo shop and suite-nonsplit and almost all of them support `~4.3` version. Sorry, I should've check before writing a comment. I think we should go with 4.3.81

**Review by pavelmaksimov25 — DISMISSED** [review]

**Review by pavelmaksimov25 — DISMISSED** [review]

**Review by pavelmaksimov25 — DISMISSED** [review]

**Review by Kyrylo-Meretskyi — APPROVED** [review]

**Review by pavelmaksimov25 — APPROVED** [review]

**Review by pavelmaksimov25 — APPROVED** [review]

**Pull request merged**